Gems/EditorPythonBindings/Code/Tests/PythonThreadingTests.cpp

```diff
@@ -114,38 +114,29 @@ namespace UnitTest
         try
         {
             // prepare handler on this thread
-            pybind11::exec(R"(
-                import azlmbr.test
-
-                def on_notification(args):
-                    value = args[0] + 2
-                    print ('RanInThread')
-                    return value
-
-                handler = azlmbr.test.PythonThreadNotificationBusHandler()
-                handler.connect()
-                handler.add_callback('OnNotification', on_notification)
-                )");
+            const AZStd::string_view script =
+                "import azlmbr.test\n"
+                "\n"
```

Comment:
just curious why the string was taken out of raw string format

Reply:
I tried to use the raw string format, but when used with `EditorPythonRunnerRequests::ExecuteByString` it doesn't work unfortunately :(
With raw format all the lines have many spaces before them, which seems to mess up the script execution, it's interesting that `pybind11::exec` takes it fine.
If you know a way to make it work work with `ExecuteByString` please let me know.